src/components/AddItemForm.js

```diff
@@ -78,7 +87,7 @@ class AddItemForm extends React.Component {
       <>
         <form onSubmit={this.handleSubmit}>
           <label className="name">
```

Comment:
When I ran the accessibility checker I got the alert that the label is not associated with the input. To fix this we need a `for=" "` attribute on the `<label>` and and `id=" "` attribute on the `<input>`

Example:
```<label className="name" for="name">
<input
              id="name"
              type="text"
              value={this.state.name}
              onChange={this.handleChange}
            />
```

Reply:
True. But in JSX it's htmlFor.